service/kvmconfig/v4/resource_set.go

```diff
@@ -37,8 +37,7 @@ type ResourceSetConfig struct {
 
 	GuestUpdateEnabled    bool
 	HandledVersionBundles []string
-	// Name is the project name.
-	Name string
+	ProjectName           string
```

Comment:
Boyscouting: this is on par with aws-operator. I changed only in v4 it will be eventually everywhere.